aloscene/depth.py

```diff
@@ -22,18 +24,55 @@ class Depth(aloscene.tensors.SpatialAugmentedTensor):
     """
 
     @staticmethod
-    def __new__(cls, x, occlusion: Mask = None, *args, names=("C", "H", "W"), **kwargs):
+    def __new__(
+            cls, 
+            x, 
+            occlusion: Mask = None, 
+            is_inverse=False, 
+            scale=None, 
+            shift=None, 
+            *args, 
+            names=("C", "H", "W"), 
+            **kwargs):
+        if is_inverse and not (shift and scale):
+            raise AttributeError('inversed depth requires shift and scale arguments')
         if isinstance(x, str):
             x = load_depth(x)
             names = ("C", "H", "W")
         tensor = super().__new__(cls, x, *args, names=names, **kwargs)
         tensor.add_child("occlusion", occlusion, align_dim=["B", "T"], mergeable=True)
-
+        tensor.add_property('_scale', scale)
+        tensor.add_property('_shift', shift)
+        tensor.add_property('_is_inverse', is_inverse)
         return tensor
 
     def __init__(self, x, *args, **kwargs):
         super().__init__(x)
 
+    def encode_inverse(self, scale: float, shift: float):
```

Comment:
I think that scale & shift should be optional when doing the inverse. By default, encode inverse should only do 1/d

Reply:
scale=1 and shift=0 by default